main/helper.py

```diff
@@ -0,0 +1,39 @@
+
+from .models import *
```

Comment:
Please don't use `import *` in code, it's only for quickly trying out features in the Python shell. See https://www.python.org/dev/peps/pep-0008/#imports for reasoning.